src/main/main.ts

```
@@ -1,9 +1,9 @@
-import { app, BrowserWindow, ipcMain, shell, Menu } from "electron";
+import { app, BrowserWindow, ipcMain, shell, Menu, WebContents, ipcRenderer } from "electron";
```

ESLint

[error] Remove this unused import of 'WebContents'.

(sonarjs/unused-import)